### PR TITLE
Goreleaser and Travis setup

### PR DESCRIPTION
**Pull request by emilingerslev**
Add minimal setup of goreleaser and travis﻿

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+builds:
+  - env:
+      - CGO_ENABLED=0
+    ldflags:
+      - "-s -w -X github.com/lunarway/strong-duckling/cmd.version={{.Version}} -X github.com/lunarway/strong-duckling/cmd.commit={{.Commit}}"
```
Comment (Crevil) [inline .goreleaser.yml]: We only have the version field available right now.
```suggestion
      - "-s -w -X github.com/lunarway/strong-duckling/cmd.version={{.Version}}"
```

**Comment by emilingerslev** [comment]
Oh, didn't see your addtional fix. What the purpose of this?

**Comment by Crevil** [comment]
Actually I don't think this is correct either `-X github.com/lunarway/strong-duckling/cmd.version`.
But I'll just test it right away.

**Comment by Crevil** [comment]
It is the build flags that populates the global version (and commit in shuttle) variables in main at build time.

**Comment by emilingerslev** [comment]
Ok. Its copied directly from the one used in shuttle, so it should work. But maybe we can discuss the problems with the current config?

**Comment by emilingerslev** [comment]
Hmm, alright, and with the wrong configuration the variable just won't be seeing correctly?

**Comment by Crevil** [comment]
Yeah, it will stay empty.

**Comment by Crevil** [comment]
#5 fixes this, jut for reference

**Pull request merged**